streams/src/main/java/org/apache/kafka/streams/state/BuiltinDslStoreSuppliers.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.state;
+
+import org.apache.kafka.streams.kstream.EmitStrategy;
+import org.apache.kafka.streams.state.internals.RocksDbIndexedTimeOrderedWindowBytesStoreSupplier;
+import org.apache.kafka.streams.state.internals.RocksDbTimeOrderedSessionBytesStoreSupplier;
+
+public class BuiltinDslStoreSuppliers {
+
+    public static final DslStoreSuppliers ROCKS_DB = new RocksDbDslStoreSuppliers();
+    public static final DslStoreSuppliers IN_MEMORY = new InMemoryDslStoreSuppliers();
+
+    public static class RocksDbDslStoreSuppliers implements DslStoreSuppliers {
```

Comment:
another naming nit: we always capitalize the "B" in "RocksDB" in class names

Reply:
made the change, but FWIW it doesn't seem consistent throughout the code base 
![image](https://github.com/apache/kafka/assets/3172405/b2a083ab-9f33-4044-aefc-60ed1b3b49ee)

Reply:
Ah, fair enough...well just personally I think it looks better capitalized but I guess we should just make a pass to converge all the class names one way or another. Not a big deal at all though